addons/website/models/res_config_settings.py

```diff
@@ -23,8 +23,7 @@ def _default_website(self):
     website_logo = fields.Binary(related='website_id.logo', readonly=False)
     language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang', readonly=False)
     website_language_count = fields.Integer(string='Number of languages', compute='_compute_website_language_count', readonly=True)
-    website_default_lang_id = fields.Many2one(string='Default language', related='website_id.default_lang_id',
-                                              readonly=False, relation='res.lang')
+    website_default_lang_id = fields.Many2one(string='Default language', related='website_id.default_lang_id', readonly=False)
```

Comment:
The alternative is to pass the comodel name as first positional argument
```suggestion
    website_default_lang_id = fields.Many2one('res.lang', string='Default language', related='website_id.default_lang_id', readonly=False)
```